src/controllers/userController.js

```diff
@@ -0,0 +1,153 @@
+import { userServices } from './../services/userService.js';
+import { v4 as uuidv4 } from 'uuid';
+
+let user = '';
+
+const defaultChatId = uuidv4();
+
+const rooms = [
+  {
+    id: defaultChatId,
+    creator: '',
+    name: 'Main-room',
+    history: [],
+  },
+];
+
+function messageValidation(message) {
+  if (!message || message.trim().length === 0) {
+    return 'Message required';
+  }
+
+  if (message.trim().length > 3000) {
+    return 'Message is to long';
```

Comment:
There is a typo in the error message: 'Message is to long' should be 'Message is too long'.